flight/utils/movement_controller.py

```diff
@@ -203,11 +203,9 @@ async def move_to_takeoff(self, drone: System, takeoff_location: LatLon) -> None
             # if the x and y values are close enough (2m) to the original position * precision
             # if inside the circle, move on to the next
             # if outside of the circle, keep running to you get inside
-            if (
-                abs(x) <= reference_x * config.POINT_PERCENT_ACCURACY
-                and abs(y) <= reference_y * config.POINT_PERCENT_ACCURACY
-            ):
+            if abs(x) <= reference_x * 0.15 and abs(y) <= reference_y * 0.15:
```

Comment:
I understand what you are doing here, but on the physical drone having to high of point accuracy causes the drone to "miss" its target and fly back and forth like it did at the flight test while trying to land.